### PR TITLE
[LIBS-128] Update version to 2.0.1, upstream merge

### PR DESCRIPTION
**Pull request by cesargdm**

### CODE REVIEW

File: packages/react-scripts/package.json [hunk 1]
```diff
@@ -27,79 +27,80 @@
...
+    "eslint-plugin-react": "7.12.4",
+    "eslint-plugin-react-hooks": "1.5.0",
+    "file-loader": "3.0.1",
+    "fs-extra": "7.0.1",
+    "html-webpack-plugin": "4.0.0-beta.5",
     "identity-obj-proxy": "3.0.0",
-    "jest": "23.6.0",
-    "jest-pnp-resolver": "1.0.1",
-    "jest-resolve": "23.6.0",
-    "lint-staged": "7.3.0",
```
Comment (dyegolara) [inline packages/react-scripts/package.json]: we need `lint-staged` for this package to work

File: packages/react-scripts/package.json [hunk 2]
```diff
@@ -27,79 +27,80 @@
...
     "eslint-plugin-flowtype": "2.50.1",
-    "eslint-plugin-import": "2.14.0",
+    "eslint-plugin-import": "2.16.0",
     "eslint-plugin-jsx-a11y": "6.2.1",
-    "eslint-plugin-react": "7.11.1",
-    "file-loader": "2.0.0",
-    "fork-ts-checker-webpack-plugin-alt": "0.4.14",
-    "fs-extra": "7.0.0",
-    "html-webpack-plugin": "4.0.0-alpha.2",
-    "husky": "1.1.0",
```
Comment (dyegolara) [inline packages/react-scripts/package.json]: also need `husky` pls

File: packages/react-scripts/package.json [hunk 3]
```diff
@@ -27,79 +27,80 @@
...
+    "file-loader": "3.0.1",
+    "fs-extra": "7.0.1",
+    "html-webpack-plugin": "4.0.0-beta.5",
     "identity-obj-proxy": "3.0.0",
-    "jest": "23.6.0",
-    "jest-pnp-resolver": "1.0.1",
-    "jest-resolve": "23.6.0",
-    "lint-staged": "7.3.0",
-    "mini-css-extract-plugin": "0.4.3",
-    "node-sass": "4.9.3",
```
Comment (dyegolara) [inline packages/react-scripts/package.json]: we also need `node-sass` for our apps build

File: packages/react-scripts/package.json [hunk 4]
```diff
@@ -27,79 +27,80 @@
...
     "optimize-css-assets-webpack-plugin": "5.0.1",
-    "pnp-webpack-plugin": "1.1.0",
+    "pnp-webpack-plugin": "1.2.1",
     "postcss-flexbugs-fixes": "4.1.0",
     "postcss-loader": "3.0.0",
-    "postcss-preset-env": "6.3.1",
+    "postcss-normalize": "7.0.1",
+    "postcss-preset-env": "6.6.0",
     "postcss-safe-parser": "4.0.1",
-    "prettier": "1.15.3",
```
Comment (dyegolara) [inline packages/react-scripts/package.json]: also `prettier`

File: packages/react-scripts/package.json [hunk 5]
```diff
@@ -27,79 +27,80 @@
...
-    "react-dev-utils": "^6.1.1",
-    "resolve": "1.8.1",
+    "react-app-polyfill": "1.0.0",
+    "react-dev-utils": "9.0.0",
+    "resolve": "1.10.0",
     "sass-loader": "7.1.0",
-    "style-loader": "0.23.0",
-    "stylelint": "9.10.1",
-    "stylelint-config-standard": "18.2.0",
-    "stylelint-scss": "3.5.3",
```
Comment (dyegolara) [inline packages/react-scripts/package.json]: also all of the `stylelint` packages

File: packages/eslint-config-react-app/index.js [hunk 6]
```diff
@@ -234,6 +278,9 @@ module.exports = {
     'jsx-a11y/role-supports-aria-props': 'warn',
     'jsx-a11y/scope': 'warn',
 
+    // https://github.com/facebook/react/tree/master/packages/eslint-plugin-react-hooks
+    'react-hooks/rules-of-hooks': 'error',
```
Comment (dyegolara) [inline packages/eslint-config-react-app/index.js]: Can we add this rule to our `eslint` pls, It wont take it from here

File: packages/react-scripts/package.json (hunk 3 above)
Comment (jmz7v) [reply packages/react-scripts/package.json]: With the setup for post css is node-sass still needed?

Comment (dyegolara) [reply packages/react-scripts/package.json]: All of our apps still use sass, only ui-components are using pcss

File: packages/react-scripts/package.json (hunk 4 above)
Comment (dyegolara) [reply packages/react-scripts/package.json]: @cesargdm we need prettier, pls

**Pull request merged**